gen3cirrus/aws/utils.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import json
+
+import boto3
+from botocore.exceptions import ClientError
+
+from cdislogging import get_logger
+
+logger = get_logger(__name__, log_level="info")
+
+
+def generatePresignedURL(
+    client, method, bucket_name, object_name, expires, additional_info={}
+):
+    """
+    Function for generating a presigned URL for upload or download
+
+    Args:
+        client: s3 boto client
+        method: ["get", "put"] "get" for download and "put" for upload
+        bucket_name: s3 bucket name
+        object_name: s3 bucket object key
+        expires: time for presigned URL to exist (in seconds)
+        additional_info: dict of additional parameters to pass to s3 for signing
+    """
+
+    params = {}
+    params["Bucket"] = bucket_name
+    params["Key"] = object_name
+
+    for key in additional_info:
+        params[key] = additional_info[key]
+
+    s3_client = client
+
+    if method == "get":
+        m = "get_object"
```

Comment:
I'd prefer not to have single character names outside of simple loops. perhaps use `method` or something similar